README.md

```
@@ -1,2 +1,75 @@
+# Raytracer :white_circle: :sunny:
+Raytracer is a project we've done at Epitech. The goal of this project is simple : do a raytracer.
+
+## Setup 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)